save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/SandboxView.kt

```diff
@@ -208,6 +226,12 @@ class SandboxView : AbstractView<SandboxViewProps, SandboxViewState>(true) {
                     FileType.SETUP_SH -> state.setupShText
                     else -> "Press on any of the buttons above to start editing"
```

Comment:
Is this branch reachable? I think we can always initialize `selectedFile` with a certain tab

Reply:
We can, but it might be not that obvious that there are three tabs. The reason FileType is nullable here is that I wanted to tell user that there is an ability to select the file he wants to delete. Moreover, it is `when` with no default case, I guess diktat would like to add `else`

Reply:
> to tell user that there is an ability to select the file

OK, makes sense